examples/service-consumer-app/src-tauri/gen/android/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         google()
+        mavenLocal()
```

Comment:
Not first here?

Reply:
yeah should probably make them consistent, fixed: https://github.com/holochain/android-service-runtime/pull/53/commits/3d31d968a0ba1f87b169a49adcd05f248732968f